frontend/class-public.php

```diff
@@ -71,6 +71,29 @@ public function enqueue_styles() {
 	}
 
 	/**
+	 * Hide the honeypot field for the volunteer sign up form.
+	 *
+	 * We load this CSS separately to be sure the field is hidden even if the 
+	 * admin has turned off loading the CSS within the settings.
+	 */
+	public function enqueue_honeypot_styles(){
+
+		if( is_singular( 'volunteer_opp' ) ): ?>
+		
+		<style>
+			/* Hide the Wired Impact Volunteer Management honeypot field under all circumstances */
+			.wivm_hp { 
+				display: none !important;
```

Comment:
As we talked about in chat, spambots are better at reading inline styling than at connecting a class to a field. I read something that said if you use a class the spambot basically must have a CSS parser, which is pretty complicated to build. To look at inline styling and determine if the field is hidden is much easier.  Great suggestion though and I wish we could go that route. It would have been much simpler and cleaner.